_posts/en/newsletters/2019-07-24-newsletter.md

```diff
@@ -0,0 +1,139 @@
+{% include references.md %}
+
+## Action items
+
+- **Help test Bitcoin Core 0.18.1 release candidates:** this upcoming
+  maintenance release fixes several bugs that affected certain RPC
+  commands and caused unnecessarily high CPU use in certain cases.
+  Production users are encouraged to test the current [release
+  candidate][core rc] to ensure that it operates as expected.
+
+## News
+
+- **Progress on signet:** signet is a testnet alternative where valid
+  blocks are signed by a trusted authority that generally produces
+  a regular series of blocks but which may occasionally produce forks leading
+  to block chain reorganizations.  This avoids commonly-encountered
+  problems on testnet such as too-fast block production, too-slow block
+  production, and reorganizations involving thousands of blocks.
+  Subsequent to our earlier report on signet in [Newsletter #37], its
+  author has created an implementation documented on a [wiki
+  page][signet wiki], opened a [PR][Bitcoin Core #16411] proposing to
+  add support for signet to Bitcoin Core, and posted a [draft
+  BIP][signet bip post] to the Bitcoin-Dev mailing list.  Signet has the
+  potential to make it much easier for developers to test their
+  applications in a multi-user environment, so we encourage all current
+  testnet users and anyone else interested in the system to review the
+  above code and documentation to ensure signet will fulfill your needs.
+
+- **Additional Just-In-Time (JIT) LN routing discussion:** in the JIT
+  discussion described in [last week's newsletter][Newsletter #55],
+  contributor ZmnSCPxj explained why routing nodes needed zero-fee
+  rebalance operations in many cases for JIT routing to be incentive
+  compatible.  This week he [posted][jit with fee] a suggestion to the C-Lightning
+  mailing list on how nodes could be less incentive compatible but still
+  prevent abuse when performing paid rebalancing for JIT routing.
+  Nodes would keep track of how much routing fee they had earned from each
+  channel and spend up to that amount on rebalancing.  This would ensure
+  that a dishonest channel counterparty couldn't steal any more than it
+  had already allowed its honest peers to claim in routing fees.
+
+## Bech32 sending support
+
+*Week 19 of 24 in a [series][bech32 series] about allowing the people
+you pay to access all of segwit's benefits.*
+
+{% comment %}<!-- weekly reminder for harding: check Bech32 Adoption
+wiki page for changes -->{% endcomment %}
+
+{% include specials/bech32/19-real-fees.md %}
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #15891][] changes the node defaults when using the
```

Comment:
s/when using the regtest chain/when running in regtest mode/ (since there isn't a single regtest chain).